web/src/app/lists/QueryList.tsx

```diff
@@ -76,6 +76,7 @@ export interface QueryListProps extends ControlledPaginatedListProps {
   // note: The `input.search` and `input.first` parameters are included by default, but can be overridden
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   variables?: any
+  mapVariables?: (vars: any) => any
```

Comment:
Need to add the correct type (`OperationVariables`) and add a comment for what this prop does.